google/auth/pluggable.py

```diff
@@ -301,11 +301,14 @@ def _parse_subject_token(self, response):
                     response["code"], response["message"]
                 )
             )
-        if "expiration_time" not in response:
+        if (
+            "expiration_time" not in response
+            and self._credential_source_executable_output_file
+        ):
             raise ValueError(
-                "The executable response is missing the expiration_time field."
+                "The output file response must contain the expiration_time field when success=true."
```

Comment:
This isn't only the output file response, it could also be the response to STDOUT. Need to update this

"The response must contain an expiration_time for successful responses when an output_file has been specified in the configuration."